### PR TITLE
switch terminology (instant app vs. quickstart)

### PR DESCRIPTION
**Pull request by gabemontero**
@adellape PTAL - I'm pretty sure this is not the only place where InstantApp vs. Quickstart terminology conversions need to occur, but since I know I created this content for the online dev preview (based on input from @sspeiche) just prior to @bparees 's work with the quickstart tutorial, I knew this at least had to be changed :-)

And of course comments from one and all for further massaging of this content always welcome.

### CODE REVIEW

File: getting_started/developers/developers_cli.adoc [hunk 1]
```diff
@@ -36,8 +36,8 @@ ifdef::openshift-online[]
 That said, similar to version 2, OpenShift provides out of the box a set of
 languages and link:../../using_images/db_images/index.html[databases] for
 developers with corresponding implementations and tutorials that allow you to
-kickstart your application development. Language support center around the
-link:../../dev_guide/templates.html#using-the-instantapp-templates[Instant App templates], 
+kickstart your application development. Language support centers around the
+link:../../dev_guide/app_tutorials/quickstarts.html#using-the-instantapp-templates[Quickstart templates],
```
Comment (bparees) [inline getting_started/developers/developers_cli.adoc]: Is that anchor right? It still says instant app. If it is right, can you change the actual anchor and then update this?

Comment (gabemontero) [inline getting_started/developers/developers_cli.adoc]: Nope that anchor is wrong (I should have removed that anchor when I updated
the main link)

**Comment by gabemontero** [comment]
removal of anchor pushed

**Comment by bparees** [comment]
lgtm.
@openshift/team-documentation please review/merge

**Comment by adellape** [comment]
:+1:

**Pull request merged**